django_project/certification/views/certificate.py

```diff
@@ -948,8 +956,16 @@ def preview_certificate(request, **kwargs):
 
         current_site = request.META['HTTP_HOST']
 
-        generate_pdf(
-            response, project, course, attendee, certificate, current_site)
+        if certificate_type_id:
+            certificate_type = CertificateType.objects.get(
```

Comment:
@sumandari please add a check here

Reply:
I have updated it with try catch block for this get query